aqueduct_client/aqueduct_client.py

```diff
@@ -306,3 +337,13 @@ def _post(self, path, body):
             headers={'Quantopian-API-Key': self._api_key},
             json=body,
         )
+
+    def _retry(self, func, timeout_seconds, *args, **kwargs):
+        retry = Retrying(
```

Comment:
FYI, I believe `requests` (the library we're using for these HTTP requests) already has a relatively well-supported interface for configuring retries. c.f. https://2.python-requests.org/en/master/api/#requests.adapters.HTTPAdapter and https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#urllib3.util.retry.Retry (`Retry` is accepted in place of an integer for `max_retries`).

Reply:
Oh cool - didn't realize it had `backoff_factor`